src/connectedk8s/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Ignore user-specific test configuration
+azext_connectedk8s/tests/latest/config.json
```

Comment:
Why do you need to submit this change?

Reply:
Previously, test code was using hard-coded user-specific configs. Updated to use config.json to be more secure. Basically, a tester fills out the user-specific settings in config.json and then run the test, whereas before it is directly put in the test code.

Reply:
Got it, thanks~